live-examples/js-examples/dataview-byteoffset.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js" class="language-js">//create an ArrayBuffer
+var buffer = new ArrayBuffer(16);
+
+// Create a couple of views
+var view1 = new DataView(buffer);
+var view2 = new DataView(buffer,12,4); //from byte 12 for the next 4 bytes
+
+console.log(view1.byteOffset + view2.byteOffset); // 0 + 12
```

Comment:
Adding offsets together seems like a slightly bizarre thing to do to me - would you ever really do that? I'd just show the two offsets console.log'ed separately - this would indicate a comparison of the offsets of two different views, which would make more sense.

Reply:
Hi Chris :) It is odd - agreed. But we're working on the assumption that we can only have one console.log for now (we've encountered issues with multiple console logs).

That said I think there is a case for simplifying here and just having view2 and its offset.

Reply:
OK, fair enough — I wasn't aware of that limitation. Your simplification sounds good.

Reply:
OK will make that change 👍

Reply:
@maboa That limitation is now removed with https://github.com/mdn/interactive-examples/pull/245 :)